packages/react/src/DescriptionList/DescriptionListDetails.test.tsx

```diff
@@ -3,7 +3,7 @@ import { createRef } from 'react'
 import { DescriptionList } from './DescriptionList'
 import '@testing-library/jest-dom'
 
-describe('Description list details', () => {
+describe('Description List Description', () => {
```

Comment:
This means the subcomponent should be called `DescriptionList.Description` as well, no?

Reply:
Yes, happy to create a PR after merging this one.